packages/realm-server/tests/loader-test.ts

```diff
@@ -178,4 +178,28 @@ module('loader', function (hooks) {
     let testingLoader = Loader.getLoaderFor(card);
     assert.strictEqual(testingLoader, loader, 'the loaders are the same');
   });
+
+  test('is able to follow redirects', async function (assert) {
+    loader.prependURLHandlers([
+      async (request) => {
+        if (request.url.includes('node-b.abc')) {
```

Comment:
whats the abc. Is it clearer if its node-b.ab??

Reply:
.abc is just a url domain I made up.